package day44_Exceptions;

public class try_catchBlock2 {
    public static void main(String[] args) {
        System.out.println(("Hello"));


        try {
            Thread.sleep(2000);
            System.out.println("try block");
        }catch (Exception e){


        }

        System.out.println("World!");
    }
}
